spec/models/publication_spec.rb

```diff
@@ -346,19 +346,25 @@
   end
 
   describe 'update_manual_pub_from_pub_hash' do
+    let(:pub) { Publication.build_new_manual_publication({ a: :b }, 'some string') }
+
     it 'should update the user submitted source record with the new content' do
-      pub = Publication.build_new_manual_publication({ a: :b }, 'some string')
       pub.update_manual_pub_from_pub_hash({ b: :c }, 'some other string')
       pub.save!
       expect(pub.user_submitted_source_records.first[:source_data]).to eq('some other string')
       expect(pub.pub_hash).to include(b: :c)
     end
 
     it 'should raise an exception if you try to update a record to match an existing source record' do
-      Publication.build_new_manual_publication({ a: :b }, 'some string').save!
-      pub = Publication.build_new_manual_publication({ b: :c }, 'some other string')
-      pub.update_manual_pub_from_pub_hash({ b: :c }, 'some string')
-      expect { pub.save! }.to raise_error(ActiveRecord::RecordNotUnique)
+      pub.save!
+      other = Publication.build_new_manual_publication({ b: :c }, 'some other string')
+      other.update_manual_pub_from_pub_hash({ b: :c }, 'some string')
+      expect { other.save! }.to raise_error(ActiveRecord::RecordNotUnique)
+    end
```

Comment:
:+1: - the use of `other` makes it clear, although what's not entirely clear is why the `other` now matches `pub` after the update.  If `some string` is a particular field on the model, it seems fair to create a `let` for it that uses the field label; e.g., if the field is title, a `let(:title) { 'some title' }` would help to clarify what this is about.